### PR TITLE
Quote ctl arguments before passing to omnibus-ctl

### PR DESCRIPTION
**Pull request by jeremiahsnapp**

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ sudo: false
 branches:
   only:
     - master
+    - byop-phase-1
```
Comment (smith) [inline .travis.yml]: does this need to be here?

Comment (marcparadise) [inline .travis.yml]: It did in order to trigger the travis build of this PR.  We can remove this commit from the PR now though - we're just about done with the branch at this point.

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

**Comment by jeremiahsnapp** [comment]
I just removed the commit that triggered the travis run. Sorry if that was the wrong thing to do. It appears that we've lost the record that the travis run was successful.

**Comment by jeremiahsnapp** [comment]
@marcparadise Can we get this added to byop-phase-1 PR please?

**Comment by marcparadise** [comment]
Would you be able to open the PR against that branch?

**Comment by jeremiahsnapp** [comment]
byop-phase-1 branch has already been merged so i'm going to open a new PR against master

**Pull request closed without merging**